basic-javascript/tests/project-1.test.js

```diff
@@ -10,178 +11,256 @@ const funcs = require('../src/project-1');
 
 describe('Project-1 Functions', () => {
   describe('`multiplyByTen`', () => {
+    const multiplyByTen = funcs.multiplyByTen;
```

Comment:
Awesome! I like that you're re-using this variable 👌

Reply:
You can use destructuring assignment here to save yourself even more typing!

`const multiplyByTen = funcs.multiplyByTen;`

Reply:
You can use destructuring assignment here to save yourself even more typing!

`const { multiplyByTen } = funcs.multiplyByTen;`